docs/SPIR-V.rst

```diff
@@ -833,6 +833,10 @@ some system-value (SV) semantic strings will be translated into SPIR-V
 | SV_RenderTargetArrayIndex +-------------+--------------------------+-----------------------+
 |                           | PSIn        | ``Layer``                | N/A                   |
 +---------------------------+-------------+--------------------------+-----------------------+
+|                           | GSOut       | ``ViewportIndex``        | N/A                   |
```

Comment:
I wonder if we need to add a column to this table for required capabilities for each semantic?
e.g. MultiViewport in this case. (some of them require Geometry, Tessellation, etc)

Reply:
Good point. The next commit is also for adding SV semantics support. I'll do it in the next commit to save Appveyor some build time. :)